planning/behavior_velocity_planner/src/scene_module/speed_bump/scene.cpp

```diff
@@ -0,0 +1,207 @@
+// Copyright 2020 Tier IV, Inc., Leo Drive Teknoloji A.Ş.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "scene_module/speed_bump/scene.hpp"
+
+#include "motion_utils/motion_utils.hpp"
+#include "scene_module/crosswalk/util.hpp"
+#include "tier4_autoware_utils/tier4_autoware_utils.hpp"
+
+#include <rclcpp/rclcpp.hpp>
+
+namespace behavior_velocity_planner
+{
+using motion_utils::calcLongitudinalOffsetPoint;
+using tier4_autoware_utils::createPoint;
+
+namespace
+{
+}  // namespace
+
+SpeedBumpModule::SpeedBumpModule(
+  const int64_t module_id, const int64_t lane_id,
+  const lanelet::autoware::SpeedBump & speed_bump_reg_elem, const PlannerParam & planner_param,
+  const rclcpp::Logger & logger, const rclcpp::Clock::SharedPtr clock)
+: SceneModuleInterface(module_id, logger, clock),
+  module_id_(module_id),
+  lane_id_(lane_id),
+  speed_bump_reg_elem_(std::move(speed_bump_reg_elem)),
+  planner_param_(planner_param),
+  state_(State::SLOW_DOWN),
+  passed_slow_start_point_(false)
+{
+  // Read speed bump height [m] from map
+  const auto speed_bump_height =
+    static_cast<float>(speed_bump_reg_elem_.speedBump().attributeOr("height", 0.5));
+
+  // If slow_down_speed is specified on speed_bump annotation use it instead of calculating it
+  if (!speed_bump_reg_elem_.speedBump().hasAttribute("slow_down_speed")) {
```

Comment:
When I read the comment, I rather expect if-else condition the other way around.

```
  if (speed_bump_reg_elem_.speedBump().hasAttribute("slow_down_speed")) {
      speed_bump_slow_down_speed_ = static_cast<float>(
      speed_bump_reg_elem_.speedBump().attribute("slow_down_speed").asDouble().get() / 3.6);
  } else {
      // calculation of slow down speed from height
  }

Reply:
You are right! I will update the comment. Thank you.